### PR TITLE
New release step.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: ReleaseChecklist.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Checklist for making a release:
  - [ ] Run ``scripts/release_ppa.sh release`` to create the PPA release (you need the relevant openssl key).
  - [ ] Check that the Docker release was pushed to Docker Hub (this still seems to have problems).
  - [ ] Update the homebrew realease in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb (version and hash)
+ - [ ] Update the default version on readthedocs.
```
Comment (axic) [inline ReleaseChecklist.md:14]: Do you mean to update it for every single example ?

Comment (chriseth) [reply ReleaseChecklist.md:14]: No, it's a setting in the UI of readthedocs.

Comment (chriseth) [reply ReleaseChecklist.md:14]: Basically the setting is where `solidity.readthedocs.io` points.

Comment (axic) [reply ReleaseChecklist.md:14]: Ah cool, so far it was set to `develop` and you've fixed it to a released version.

**Pull request merged**